Update README.md (#5393)
Qiskit/documentation: README.md

   3. Click the CI run name. (Not the middle column with the blue link!)
   4. In the left navbar, it should show as selected the "Summary" page with the house.
   5. Scroll down to "Artifacts" and look for the artifact related to documentation, such as `html_docs`.

>    6. Download the artifact by clicking on its name. **Note:** For `qiskit-addon-utils` (and possibly others) the downloaded zip file might contain an extra parent directory. This will cause the `gen-addon` script to run incorrectly and leave your changes in a broken state. To fix this, uncompress the zip file and open the top-most directory, so that you are viewing a collection of files and folders such as `_downloads`, `_images`, `objects.inv`, and others. Select all these files and folders and compress into a new zip file, which you will now rename (see step 5 below) and upload (step 6).

Reviewer: ```suggestion
   6. Download the artifact by clicking on its name. **Note:** For `qiskit-addon-utils` (and possibly others) the downloaded zip file might contain an extra parent directory. This will cause the `gen-addon` script to run incorrectly and leave your changes in a broken state. To fix this, uncompress the zip file and open the top-level directory, so you are viewing a collection of files and folders such as `_downloads`, `_images`, `objects.inv`, and others. Select all these files and folders and compress into a new zip file, which you will rename (see step 5 below) and upload (step 6).
```

>    6. Download the artifact by clicking on its name. **Note:** For qiskit-addon-utils (and possibly others?) the downloaded zip file contains an extra parent directory, meaning that the step to copy over the docs for the newest version will only delete files, and not add or change existing ones. To fix this, uncompress the zip file and open the top-most directory, so that you are viewing a collection of files and folders such as `_downloads`, `_images`, `objects.inv`, and others. Select all these files and folders and compress into a new zip file, which you will now rename (see step 5 below) and upload (step 6).

Reviewer 2: ```suggestion
   6. Download the artifact by clicking on its name. **Note:** For `qiskit-addon-utils` (and possibly others) the downloaded zip file might contain an extra parent directory. This will cause the `gen-addon` script to run incorrectly and leave your changes in a broken state. To fix this, uncompress the zip file and open the top-most directory, so that you are viewing a collection of files and folders such as `_downloads`, `_images`, `objects.inv`, and others. Select all these files and folders and compress into a new zip file, which you will now rename (see step 5 below) and upload (step 6).
```
